fds/cli.py

```diff
@@ -29,14 +29,37 @@
                            action="store_true", default=False)
 parser_commit.add_argument('message', help="commit message")
 
+# push
+parser_push = command_subparser.add_parser(
+    'push',
+    help='push commits to remote git and dvc repository'
+)
+parser_push.add_argument('-gr', '--git-remote', help="git remote name, default 'origin'", default="origin")
+parser_push.add_argument('-dr', '--dvc-remote', help="dvc remote name, default 'origin'", default="origin")
+parser_push.add_argument(
+    'branch',
+    help="A git branch that you want to push to (accepts refspec), defaults to current branch",
+    nargs='?'
+)
+# save
+parser_save = command_subparser.add_parser(
+    'save',
+    help='saves all project files to a new version and pushes them to your remote'
+)
+parser_save.add_argument('-gr', '--git-remote', help="git remote name, default 'origin'", default="origin")
+parser_save.add_argument('-dr', '--dvc-remote', help="dvc remote name, default 'origin'", default="origin")
```

Comment:
Maybe add a flag for the branch name?

Reply:
I want it to be simple. It's probably also a bad idea to have these flags. I'm actually thinking of removing them, once we implement the "check if there is a remote added for both, then just run it" it has to be dead simple

Reply:
The branch flag is a nice-to-have since 90% of the time it will be the same branch you're currently on.
But I strongly disagree about removing these flags - I think you'll make the tool unusable in many cases, and that whoever doesn't want to change the remote names, doesn't have to

Reply:
One thing which WOULD be nice is to automatically detect the default remote based on `.dvc/config` and use that as the default value, but that's more advanced - we're working on that ability in #53